README.rst

```diff
@@ -11,10 +11,12 @@ This app requires:
     - for Django < 1.9      please use django-atris < 1.0.0
     - for Django < 1.10     please use django-atris < 1.2.0
     - for Django > 2.0.0    please use django-atris > 1.2.1
+    - for Django > 3.2      please use django-atris >= 2.0.1
 - Postgresql
 - Python:
     - for django-atris < 2.0.0  please use Python >= 2.7 or Python >= 3.4 (after Django 2)
     - for django-atris >= 2.0.0 please use Python >= 3.6
```

Comment:
i think = instead of >= might be cleaner

Reply:
yeah, good point